src/components/Tailwind/Modals/NetworkModal.tsx

```diff
@@ -111,6 +155,8 @@ const NetworkModal = ({ isVisible, mode, onDismiss }: NetworkModalProps) => {
                     console.log('mode', mode)
                     const params = NETWORK_PARAMS[key]
                     library?.send('wallet_addEthereumChain', [params, account])
+                  } else if (mode === NetworkModalMode.SecondaryBridge) {
```

Comment:
do we need the else?
```suggestion
                  } 
                  
                  if (mode === NetworkModalMode.SecondaryBridge) {
```

Reply:
Can you pls check this on #60 instead @bitcoinbrisbane ? cc: @mystbrent